bioio_ome_zarr/tests/writers/test_ome_zarr_writer_V3.py

```diff
@@ -61,7 +62,7 @@ def test_write_full_volume_and_metadata(
     data_generator: Any,
     expected_shapes: List[Tuple[int, ...]],
 ) -> None:
-    tmpdir = tempfile.mkdtemp()
+    tmpdir = tempfile.mkdtemp(suffix=".zarr")
```

Comment:
what is this for?

Reply:
The temp directory is a zarr 'file' right?  I was confused about that, so I thought making it explicit would be more clear in the future.

Reply:
nit. it might be easer to change the name "tempdir" to like 'zarr_source' or something then.